DragonBones/src/dragonBones/armature/Armature.h

```diff
@@ -13,7 +13,7 @@ class Bone;
 class Slot;
 class Animation;
 
-class Armature : public BaseObject, public IAnimateble
+class Armature : public virtual IAnimateble, public virtual BaseObject
```

Comment:
Two questions:
1. Why to add `virtual` keyword? I didn't find any `rhombus Inheritance `.
2. Why to switch the Inheritance order? 

BTW:
Typo: `IAnimateble` -> `IAnimatable`